ivy/functional/ivy/device.py

```diff
@@ -164,13 +164,16 @@ def dev_from_str(device: Union[ivy.Device, str]) -> ivy.Device:
     Parameters
     ----------
     device
-        The device string to conver to native device handle.
+        The device string to convert to native device handle.
 
     Returns
     -------
     ret
         Native device handle.
 
+    Examples:
```

Comment:
Please also provide more examples according to the points listed here (https://lets-unify.ai/ivy/deep_dive/11_docstring_examples.html), thanks!